diff --git a/sonic-frameworks-open-collab/sonic-frameworks-open-collab.spec b/sonic-frameworks-open-collab/sonic-frameworks-open-collab.spec
--- a/sonic-frameworks-open-collab/sonic-frameworks-open-collab.spec
+++ b/sonic-frameworks-open-collab/sonic-frameworks-open-collab.spec
@@ -11,1 +11,1 @@
-Version: 6.26.0
+Version: 6.27.0
@@ -62,1 +62,3 @@
+Provides: cmake(KF6Attica) = %{version}
+Provides: pkgconfig(KF6Attica) = %{version}
 Conflicts: %{_lib}KF6Attica-devel
@@ -70,2 +70,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
-rm -rf %{buildroot}/%{_libdir}/pkgconfig
@@ -80,2 +80,2 @@
-# %{_libdir}/cmake/KF6Attica
-# %{_libdir}/pkgconfig/KF6Attica.pc
+%{_libdir}/cmake/KF6Attica
+%{_libdir}/pkgconfig/KF6Attica.pc
