diff --git a/sonic-screen-library/sonic-screen-library.spec b/sonic-screen-library/sonic-screen-library.spec
--- a/sonic-screen-library/sonic-screen-library.spec
+++ b/sonic-screen-library/sonic-screen-library.spec
@@ -11,1 +11,1 @@
-Version:	6.6.5
+Version:	6.7.2
@@ -80,1 +80,3 @@
+Provides: cmake(KF6Screen) = %{version}
+Provides: pkgconfig(KF6Screen) = %{version}
 Conflicts:      %{_lib}KF6Screen-devel
@@ -64,2 +64,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
-rm -rf %{buildroot}/%{_libdir}/pkgconfig
@@ -90,2 +90,2 @@
-# %{_libdir}/cmake/KF6Screen
-# %{_libdir}/pkgconfig/*.pc
+%{_libdir}/cmake/KF6Screen
+%{_libdir}/pkgconfig/*.pc
