diff --git a/sonic-decoration/sonic-decoration.spec b/sonic-decoration/sonic-decoration.spec
--- a/sonic-decoration/sonic-decoration.spec
+++ b/sonic-decoration/sonic-decoration.spec
@@ -13,1 +13,1 @@
-Version:	6.6.5
+Version:	6.7.2
@@ -53,1 +53,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -63,1 +63,2 @@
+Provides: cmake(KDecoration3) = %{version}
 Conflicts: %{_lib}kdecorations3-devel
@@ -75,1 +75,1 @@
-# %{_libdir}/cmake/KDecoration3
+%{_libdir}/cmake/KDecoration3
