diff --git a/sonic-sysguard-library/sonic-sysguard-library.spec b/sonic-sysguard-library/sonic-sysguard-library.spec
--- a/sonic-sysguard-library/sonic-sysguard-library.spec
+++ b/sonic-sysguard-library/sonic-sysguard-library.spec
@@ -10,1 +10,1 @@
-%define processcore_major 10
+%define processcore_major 11
@@ -24,1 +24,1 @@
-Version:	6.6.5
+Version:	6.7.2
@@ -106,1 +106,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -108,1 +108,2 @@
 %files -f %{name}.lang
+%{_bindir}/ksysguard-identify
@@ -217,1 +217,2 @@
+Provides: cmake(KSysGuard) = %{version}
 Conflicts: %{_lib}KF6Libsysguard-devel
@@ -228,1 +228,1 @@
-# %{_libdir}/cmake/KSysGuard
+%{_libdir}/cmake/KSysGuard
