diff --git a/sonic-frameworks-runner/sonic-frameworks-runner.spec b/sonic-frameworks-runner/sonic-frameworks-runner.spec
--- a/sonic-frameworks-runner/sonic-frameworks-runner.spec
+++ b/sonic-frameworks-runner/sonic-frameworks-runner.spec
@@ -9,1 +9,1 @@
-Version: 6.26.0
+Version: 6.27.0
@@ -77,1 +77,2 @@
+Provides: cmake(KF6Runner) = %{version}
 Conflicts: %{_lib}KF6Runner-devel
@@ -83,1 +83,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -89,1 +89,2 @@
 %{_includedir}/KF6/KRunner
+%{_qtdir}/metatypes/qt6kf6runner_metatypes.json
@@ -92,1 +92,1 @@
-# %{_libdir}/cmake/KF6Runner
+%{_libdir}/cmake/KF6Runner
