diff --git a/sonic-interface-libraries/sonic-interface-libraries.spec b/sonic-interface-libraries/sonic-interface-libraries.spec
--- a/sonic-interface-libraries/sonic-interface-libraries.spec
+++ b/sonic-interface-libraries/sonic-interface-libraries.spec
@@ -106,1 +106,3 @@
+Provides: cmake(Plasma) = %{version}
+Provides: cmake(PlasmaQuick) = %{version}
 Conflicts:  %{_lib}Plasma-devel
@@ -132,1 +132,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -143,2 +143,2 @@
-# %{_libdir}/cmake/Plasma
-# %{_libdir}/cmake/PlasmaQuick
+%{_libdir}/cmake/Plasma
+%{_libdir}/cmake/PlasmaQuick
