diff --git a/sonic-frameworks-doctools/sonic-frameworks-doctools.spec b/sonic-frameworks-doctools/sonic-frameworks-doctools.spec
--- a/sonic-frameworks-doctools/sonic-frameworks-doctools.spec
+++ b/sonic-frameworks-doctools/sonic-frameworks-doctools.spec
@@ -66,1 +66,3 @@
+Provides: cmake(KF6DocTools) = %{version}
 Conflicts: %{_lib}KF6DocTools-devel
@@ -75,1 +75,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -90,1 +90,1 @@
-# %{_libdir}/cmake/KF6DocTools
+%{_libdir}/cmake/KF6DocTools
