diff --git a/sonic-frameworks-io-extras/sonic-frameworks-io-extras.spec b/sonic-frameworks-io-extras/sonic-frameworks-io-extras.spec
--- a/sonic-frameworks-io-extras/sonic-frameworks-io-extras.spec
+++ b/sonic-frameworks-io-extras/sonic-frameworks-io-extras.spec
@@ -12,1 +12,1 @@
-Version:	26.04.1
+Version:	26.04.2
@@ -100,1 +100,2 @@
+Provides: cmake(KioArchive6) = %{version}
 Conflicts: %{_lib}kioarchive6-devel
@@ -106,1 +106,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -144,1 +144,1 @@
-# %{_libdir}/cmake/KioArchive6
+%{_libdir}/cmake/KioArchive6
