diff --git a/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec b/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec
--- a/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec
+++ b/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec
@@ -14,1 +14,1 @@
-Version: 6.26.0
+Version: 6.27.0
@@ -74,1 +74,3 @@
+Provides: cmake(KF6CoreAddons) = %{version}
+Provides: pkgconfig(KF6CoreAddons) = %{version}
 Conflicts: %{_lib}KF6CoreAddons-devel
@@ -91,2 +91,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
-rm -rf %{buildroot}/%{_libdir}/pkgconfig
@@ -100,1 +100,2 @@
+%{_qtdir}/metatypes/qt6kf6coreaddons_metatypes.json
 %{_includedir}/KF6/KCoreAddons
@@ -103,2 +103,2 @@
-# %{_libdir}/cmake/KF6CoreAddons
-# %{_libdir}/pkgconfig/KF6CoreAddons.pc
+%{_libdir}/cmake/KF6CoreAddons
+%{_libdir}/pkgconfig/KF6CoreAddons.pc
