diff --git a/sonic-desktop-interface/sonic-desktop-interface.spec b/sonic-desktop-interface/sonic-desktop-interface.spec
--- a/sonic-desktop-interface/sonic-desktop-interface.spec
+++ b/sonic-desktop-interface/sonic-desktop-interface.spec
@@ -8,1 +8,1 @@
-Version: 6.6.5
+Version: 6.7.2
@@ -25,1 +25,1 @@
-BuildRequires: cmake(KWinDBusInterface) = %{version}
+BuildRequires: cmake(KWinX11DBusInterface) = %{version}
