diff --git a/sonic-frameworks-quick-ui/sonic-frameworks-quick-ui.spec b/sonic-frameworks-quick-ui/sonic-frameworks-quick-ui.spec
--- a/sonic-frameworks-quick-ui/sonic-frameworks-quick-ui.spec
+++ b/sonic-frameworks-quick-ui/sonic-frameworks-quick-ui.spec
@@ -9,1 +9,1 @@
-Version: 6.26.0
+Version: 6.27.0
@@ -69,1 +69,2 @@
+Provides: cmake(KF6KirigamiPlatform) = %{version}
 Conflicts: %{_lib}KirigamiPlatform-devel
@@ -75,1 +75,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -82,1 +82,2 @@
+%{_qtdir}/metatypes/qt6kirigamiplatform_metatypes.json
 %{_includedir}/KF6/Kirigami
@@ -85,1 +85,1 @@
-# %{_libdir}/cmake/KF6Kirigami*
+%{_libdir}/cmake/KF6Kirigami*
