diff --git a/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec b/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec
--- a/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec
+++ b/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec
@@ -27,5 +27,0 @@
-BuildRequires: python
-BuildRequires: python%{pyver}dist(build)
-BuildRequires: pkgconfig(python3)
-BuildRequires: cmake(Shiboken6)
-BuildRequires: cmake(PySide6)
@@ -55,1 +55,2 @@
+BuildOption:	-DBUILD_PYTHON_BINDINGS:BOOL=OFF
 BuildOption:	-DBUILD_QCH:BOOL=ON
@@ -81,8 +81,0 @@
-%package -n python-sonicframeworkscoreaddons
-Summary: Python bindings to SonicFrameworksCoreAddons
-Group: Development/Python
-Requires: %{libname} = %{EVRD}
-Conflicts: python-kcoreaddons
-
-%description -n python-sonicframeworkscoreaddons
-%summary
@@ -110,4 +110,0 @@
-%files -n python-sonicframeworkscoreaddons
-%{_libdir}/python*/site-packages/KCoreAddons.cpython-*.so
-%{_includedir}/PySide6/KCoreAddons
-%{_datadir}/PySide6/typesystems/typesystem_kcoreaddons.xml
